cli/internal/fs/turbo_json.go

```diff
@@ -556,6 +568,23 @@ func (btd *BookkeepingTaskDefinition) UnmarshalJSON(data []byte) error {
 	btd.experimental.PassthroughEnv = envVarPassthroughs.UnsafeListOfStrings()
 	sort.Strings(btd.experimental.PassthroughEnv)
 
+	if task.DotEnv != nil {
+		btd.definedFields.Add("DotEnv")
+
+		// Going to _at least_ be an empty array.
+		btd.TaskDefinition.DotEnv = make(turbopath.AnchoredUnixPathArray, 0, len(task.DotEnv))
+
+		// Port the raw globalDotEnv values in.
+		for _, dotEnvPath := range task.DotEnv {
+			if filepath.IsAbs(dotEnvPath) {
```

Comment:
Is this the case for only task dotenvs or also the global dot env?

Reply:
Follow along here: https://vercel.slack.com/archives/C02HEJASXGD/p1683801269758379

Reply:
(I locked it down to non-absolute for now; we can always relax an error in the future.)